charts/kube-prometheus-stack/values.yaml

```diff
@@ -1323,6 +1323,7 @@ prometheus-node-exporter:
   extraArgs:
     - --collector.filesystem.ignored-mount-points=^/(dev|proc|sys|var/lib/docker/.+|var/lib/kubelet/.+)($|/)
     - --collector.filesystem.ignored-fs-types=^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$
+  hostNetwork: true
```

Comment:
Shouldn't this be set to `false` to maintain the current expected behaviour? Or is your intention to alter the default configuration? If so this would affect everyone using the chart and I'm not certain of the full scope of ramifications to the existing install base.

Reply:
Default ist currently `true` in the chart of prometheus-node-exporter
https://github.com/prometheus-community/helm-charts/blob/3a31dda221ab79772b7bdbb97d472cc78601ea2b/charts/prometheus-node-exporter/values.yaml#L86

Reply:
Oh sorry, I was thinking I'm looking at the node-exporter chart instead of this one. I don't think it's necessary to add every single configuration option from dependent charts to the values file. It's not the case for grafana or kube-state-metrics

Reply:
Your are probably right. I would even take it a step further: There should be all or (preferably) none entries for subcharts in the values.yaml.
See, for new users like me it's quite misleading to only find a subset of supported variables. That's not only true for the grafana subchart, but also for prometheus for example where I had the same problem which resulted in #759.

It would be much more helpful to just define the values that are really needed and have a link to grafana's values.yaml for further reference.